server/src/main/java/org/rri/ideals/server/ManagedDocuments.java

```diff
@@ -139,7 +141,7 @@ public void updateDocument(@NotNull DidChangeTextDocumentParams params) {
       }
 
       try {
-        applyContentChangeEventChanges(doc, sortContentChangeEventChanges(contentChanges));
+        applyContentChangeEventChanges(doc, contentChanges);
```

Comment:
Does this mean changes come already sorted?

Reply:
I didn't find any information in the official documentation but it seems to be the case when I debug the application or if I look how it works on other lsp server (e.g. https://github.com/ fwcd/kotlin-language-server/blob/24c2e410e4718e79c5cbf79ec1dfe60161ff338c/server/src/main/kotlin/org/javacs/kt/SourceFiles.kt#L100)